nf_core/pipeline-template/nextflow.config

```diff
@@ -220,8 +220,15 @@ env {
     JULIA_DEPOT_PATH = "/usr/local/share/julia"
 }
 
-// Capture exit codes from upstream processes when piping
-process.shell = ['/bin/bash', '-euo', 'pipefail']
+// Set bash options
+process.shell = """\
+bash
+
+set -e # Exit if a tool returns a non-zero status/exit code
+set -u # Treat unset variables and parameters as an error
```

Comment:
`-u` and `-e` are already used by Nextflow by default. I could only find one mention of it in the documentation (https://www.nextflow.io/docs/latest/process.html#process-directive-shell), but I checked in one of my runs and it's correct

Reply:
Oh I didn't know that, thanks for pointing that out!

Reply:
Oh I didn't know that, thanks for pointing that out!